src/Sparrow/Json/BlittableJsonDocumentBuilder.cs

```diff
@@ -12,16 +13,20 @@ public sealed class BlittableJsonDocumentBuilder : IDisposable
     {
         private class GlobalPoolItem
         {
+            private static int MaxCacheSize = PlatformDetails.Is32Bits == false
+                ? 8 * 1024
+                : 2 * 1024;
```

Comment:
Should this be this high on 32 bits?
```suggestion
                : 512;
```

Reply:
Not sure if we want to be that conservative here? In other places the ratio is 1:4 so I thought that we could use that ratio here too.